test/powershell/engine/Help/HelpSystem.OnlineHelp.Tests.ps1

```diff
@@ -56,8 +56,38 @@ Describe 'Get-Help -Online opens the default web browser and navigates to the cm
     $skipTest = [System.Management.Automation.Platform]::IsIoT -or
                 [System.Management.Automation.Platform]::IsNanoServer
 
-    It "Get-Help get-process -online" -skip:$skipTest {
+    if($IsWindows)
+    {
+        $regKey = "HKEY_CURRENT_USER\Software\Microsoft\Windows\Shell\Associations\UrlAssociations\http\UserChoice"
 
+        try
+        {
+            $progId = [Microsoft.Win32.Registry]::GetValue($regKey, "ProgId", $null)
+            if($progId)
+            {
+                $browserKey = [Microsoft.Win32.Registry]::ClassesRoot.OpenSubKey("$progId\shell\open\command", $false)
+                if($browserKey)
+                {
+                    $browserExe = $browserKey.GetValue($null)
+                    if($browserExe -notmatch '.exe')
```

Comment:
Hi @adityapatwardhan. To get the default browser executable path you need to parse the output of $browserKey.GetValue($null). This would look something like this: $browserExe = (($browserKey.GetValue($null) -replace '"' ,"") -split " ")[0]. 

Also, you could potentially set $skipTest to $false by default which will simplify your logic to something like this:

$skipTest = $true
try
{
    $regKey = "HKEY_CURRENT_USER\Software\Microsoft\Windows\Shell\Associations\UrlAssociations\http\UserChoice"
    $progId = [Microsoft.Win32.Registry]::GetValue($regKey, "ProgId", $null)
    if($progId)
    {
        $browserKey = [Microsoft.Win32.Registry]::ClassesRoot.OpenSubKey("$progId\shell\open\command", $false)
        if($browserKey)
        {
            $browserExe = (($browserKey.GetValue($null) -replace '"' ,"") -split " ")[0]
            if($browserExe.EndsWith('.exe'))
            {
                $skipTest = $false
            }
        }
    }
}
catch 
{
    # ignore the failure
}